crypto/rsa/rsa_lib.c

```diff
@@ -316,7 +316,7 @@ uint16_t ossl_ifc_ffc_compute_security_bits(int n)
 
     /*
      * Look for common values as listed in standards.
-     * These values are not exactly equal to the results from the foruml� in
+     * These values are not exactly equal to the results from the formulae in
```

Comment:
Why not "formulas"?

Reply:
Ask Paul :grin: @paulidale

Reply:
I have no preference for Latin over English but neither the other way around.

Reply:
`c89`, `c99`, `c11` do not care, so this looks like more of a coding standard. However, if you try to use shell tools to search, you will find that you cannot paste `æ` into some implementations of `bash` through some emulators (like putty). This leaves maintainers/debuggers potentially handcuffed.

Reply:
There are interesting things to read on the subject, if language details interest you...

https://writingexplained.org/formulas-or-formulae-correct

According to that one, we better decide if our code scientific writing or not :wink:

Reply:
> c89, c99, c11 do not care, so this looks like more of a coding standard.

We have at least one report of a compiler emitting a warning on this.

Reply:
Anything outside the C locale is undefined.

Reply:
> 
> 
> > c89, c99, c11 do not care, so this looks like more of a coding standard.
> 
> We have at least one report of a compiler emitting a warning on this.

If my vote counted, I would suggest that it is more important to make compilers happy than any writing style guides for scientific papers.

Reply:
I would say at least some parts of the code is pretty scientific. :grin:

Reply:
> I would say at least some parts of the code is pretty scientific. :grin:

Not in an academic sense, i.e. it's not a scientific *writing* per se, it's an implementation of something that was probably a scientific writing first.  :grin:

I think we should stop this dispute here :wink:

Reply:
We favour proper English spel-ink which means -æ.